reports/ReportStructure.md

```diff
@@ -0,0 +1,18 @@
+# Structure of Reports and Explanation of Report Sections #
+### Report Name & Number ### 
+Every report has an unique number. Every report name should follow this convention:  
+{subsystem_name}:{file_name}:{file_extension}:{warning_type}:{unique_report_number}.md  
```

Comment:
I expected the unique_report_number to be unique and increasing within the file; hence, at the end, with your interpretation being unique over the whole repository, it would make sense to move this unique number to the beginning of the filename. Also with only 4 digits, we will probably not have enough if we start automating it, but for now that is fine and we change it later on easily.